pkgs/shells/fish/plugins/editor-updater.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "editor-updater";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "dudeofawesome";
+    repo = "fish-plugin-editor-updater";
+    rev = "v${version}";
+    hash = "sha256-B7Ze0a5Zp+5JVsQUOv97mKHh5wiv3ejsDhJMrK7YOx4=";
+  };
+
+  meta = with lib; {
+    description = "A Fish shell plugin to set your EDITOR if you're in a IDE's integrated terminal.";
```

See `pkgs/README.md` for style requirements in `meta.description`. This extends to all other plugins.
```suggestion
    description = "Fish shell plugin to set your EDITOR if you're in an IDE's integrated terminal";
```